Sources/NIOCertificateReloading/TimedCertificateReloader.swift

```diff
@@ -278,16 +278,49 @@ public struct TimedCertificateReloader: CertificateReloader {
         }
     }
 
-    private struct CertificateKeyPair {
+    private struct CertificateKeyPair: Hashable {
         var certificates: [NIOSSLCertificateSource]
         var privateKey: NIOSSLPrivateKeySource
     }
 
+    /// Provides information about the newly loaded certificate.
```

Comment:
```suggestion
    /// Provides information about the loaded certificate chain and private key.
```

Also, can we add docs for each of the properties?